app/src/main/res/values/strings.xml

```diff
@@ -182,7 +182,7 @@ be temporary, and you can try again later.</li>
     <string name="onboarding_turbo_mode_title">Turbo Mode helps you browse faster</string>
     <string name="onboarding_turbo_mode_body">Turbo Mode automatically blocks trackers and ads to help get you where you\'re going faster. If there\'s a speed bump – like a site that isn\'t behaving the way you expect – just turn Turbo Mode off.</string>
```

Comment:
Is it possible to use apostrophes `’` instead of the straight quotes `'`? The entities without charset utf-8 would be `&lsquo;` or `&#8216;`.